src/formatDate.js

```diff
@@ -50,7 +50,42 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const [, , , signFrom] = fromFormat;
+  const [, , , signTo] = toFormat;
+  const dateToArr = date.split(signFrom);
```

Comment:
```suggestion
  const dateArray = date.split(signFrom);
```
it's not a function, it's already an array